### DISCUSSION TITLE
Difference between extension *.duckdb and *.db

### DISCUSSION THREAD

**Discussion by lukjarych** [duckdb/duckdb#15455] [Q&A]
Hi,

power query wtihin power bi is reading *.db file, but not reading *.duckdb extension. 

What is the difference between them ? Why to use *.duckdb and why to use *.db file ?

Thank you in advance for your help,
Best
Luke

**Comment by Tishj** [comment]
There is no difference, it's just whatever you want to name your database file

I guess power BI does some inference on the filename, so you might want to rename it to `.db` for that, this has no effect on the contents of the file whatsoever
